### PR TITLE
Adds simple local polyfill for startsWith and endsWith

### PR DESCRIPTION
**Pull request by andyfleming**
Adds simple local polyfill for `startsWith` and `endsWith` to help with browser compatibility.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -103,3 +103,25 @@ export function getUrlTemplate(url, getState) {
...
+
+/**
+ * Determines whether a string ends with the characters of a specified string
+ *
+ * @param {String} str - String to check
+ * @param {String} search - String to search for at the end
+ * @returns {Boolean} result
+ */
+export function endsWith(str, search) {
+  return str.indexOf(search) === (str.length - search.length);
```
Comment (DullReferenceException) [inline src/utils.js]: You'll want to use `lastIndexOf` instead. Otherwise, `endsWith("siryessir", "sir") === false`.

Comment (andyfleming) [reply src/utils.js]: Good catch! I added a test case for it and changed the implementation of `endsWith` accordingly.

File: tests/utils.spec.js [hunk 2]
```diff
@@ -209,4 +211,47 @@ describe('Utils', () => {
...
+  });
+
+  describe('endsWith', () => {
+
+    it('returns true is search string is at end', () => {
+      expect(endsWith('example', 'le')).toBe(true);
+    });
+
+    it('handles a string with multiple occurences', () => {
+      expect(endsWith('siryessir', '')).toBe(true);
```
Comment (DullReferenceException) [inline tests/utils.spec.js]: Shouldn't the test case be `endsWith('siryessir', 'sir')?`

Comment (andyfleming) [reply tests/utils.spec.js]: Whoops! Yes, that's what I intended. Pushing an update now.

**Pull request merged**